post-docs/add-back-references.py

```diff
@@ -40,11 +42,18 @@ class GenerationType(enum.Enum):
 
 
 def download_file(url):
-    filedata = urlopen(url)
-    datatowrite = filedata.read()
-
-    with open('redirects.txt', 'wb') as f:
-        f.write(datatowrite)
+    with tempfile.TemporaryDirectory() as tmpdir:
+        temp_dir = Path(tmpdir)
+        file_name = temp_dir / "redirects.txt"
```

Comment:
The downloaded redirects.txt file will not be used - it is notpassed anywhere. I see that in case of construct_mapping method it still opens "current working dir" redirects.txt - which is not going to work - if you downliad the file here, it should be passed on to processing

Reply:
Thanks for the feedback. I realised this fault while testing and I have fixed it in the latest commit

Reply:
LGTM now!